### ISSUE TITLE
[XrdHttp-TPC] 0-size file transfer PUSH/PULL is not handled properly

### ISSUE THREAD

**Issue by ccaffy** [xrootd/xrootd#2113]
When one tries to PUSH/PULL a 0-size file via XrdHttp-TPC (PUSH/PULL), the transfer will fail after 120 seconds and the client will receive 

```
failure: Error when interacting with local filesystem: Transfer failed because no bytes have been received in 120 seconds.
```
I have to investigate, it's probably because the handles of curl are not properly created if the content size of the file to send is 0.

**Comment by amadio** [comment]
Thanks! That's perfect for me to test and fix #2060 and #2067 :-)

**Comment by amadio** [comment]
This works with CURL 8.4.0, which I have installed.
